.travis.yml

```diff
@@ -7,9 +7,12 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27  
-    - CONDA_PY=34  
-    - CONDA_PY=35  
+    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
Older versions did raise errors due to missing tokens. @jakirkham what version of conda-smithy do you have installed? (Although due to the recent improvements you should do directly to the latest version.)